wagtail/admin/templates/wagtailadmin/reports/base_report.html

```diff
@@ -0,0 +1,22 @@
+{% extends "wagtailadmin/base.html" %}
+{% load i18n wagtailadmin_tags %}
+{% block titletag %}{{ title }}{% endblock %}
+
+{% block content %}
+    {% include "wagtailadmin/shared/header.html" with title=title icon=header_icon %}
+
+    {% with page_obj as pages %}
+        <div id="page-results">
+            <div class="nice-padding">
+                {% if pages %}
+                    {% block listing %}
+                        {% include "wagtailadmin/pages/listing/_list_explore.html" %}
+                    {% endblock %}
+                    {% paginate pages base_url=request.path %}
+                {% else %}
+                    <p>{% trans 'No pages available.' %}</p>
```

Comment:
Could we reword this? Maybe something like "No pages match this report's criteria", or we could make this configurable per report type?

Reply:
This has been replaced with a block - thanks for the suggestion!